src/components/sections/section-utils/use-section-item.ts

```diff
@@ -33,6 +35,7 @@ export default function useSectionItem({
     padding: spacing,
     alignSelf: isInline ? 'flex-start' : undefined,
     ...mapToBorderRadius(theme, radiusSize, radius),
+    backgroundColor: transparent ? undefined : theme.background.level0,
```

Comment:
`backgroundColor` also accepts `'transparent'`, not a big deal tho, just an FYI

Reply:
Good point! But I'll just leave it and fix it later 😬